flocker/node/agents/functional/test_ebs.py

```diff
@@ -182,6 +188,132 @@ def test_next_device_in_use(self):
                                        {u"/dev/sdf"})
         self.assertEqual(result, u"/dev/sdg")
 
+    def test_create_volume_gold_profile(self):
```

Comment:
Most of these tests seem to just duplicate the standard interface tests. The exception is the fallback to DEFAULT on errors, which should be tested so it has coverage.